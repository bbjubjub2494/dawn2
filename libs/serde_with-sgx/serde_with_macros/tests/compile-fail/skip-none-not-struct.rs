use serde_with_macros::skip_serializing_none;

#[skip_serializing_none]
fn test() {}

fn main() {}
