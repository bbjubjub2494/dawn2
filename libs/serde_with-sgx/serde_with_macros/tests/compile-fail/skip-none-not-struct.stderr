error: The attribute can only be applied to struct or enum definitions.
 --> tests/compile-fail/skip-none-not-struct.rs:3:1
  |
3 | #[skip_serializing_none]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `skip_serializing_none` (in Nightly builds, run with -Z macro-backtrace for more info)
